qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/3-upgrade-with-workload.yaml

```diff
@@ -15,7 +15,7 @@ upgrade-tasks:
   - cephadm.shell:
       env: [sha1]
       host.a:
-        - while ceph orch upgrade status | jq '.in_progress' | grep true ; do ceph orch ps ; ceph versions ; ceph fs dump; sleep 30 ; done
+        - while ceph orch upgrade status | jq '.in_progress' | grep true && ! ceph orch upgrade status | jq '.message' | grep Error ; do ceph orch ps ; ceph versions ; ceph fs dump; ceph orch upgrade status ; sleep 30 ; done
```

Comment:
Suggest also just `exit 1` after 30 minutes too. No reason to let this run for 12h if something gets stuck.

Reply:
added a 30 minute timeout on the command